docs/src/Callout.doc.js

```diff
@@ -22,168 +47,354 @@ card(
+                message="It may take up to 10 minutes to automatically detect a newly installed tag. If you'd like to manually verify your tag, please click the Verify Tag button."
+                primaryAction={{label: "Verify Tag", accessibilityLabel:""}}
+                dismissButton={{
+                  accessibilityLabel: 'Dismiss this banner',
+                  onDismiss: ()=>{},
+                }}
+              />
+            </Flex>
+          </Box>
+          <Box height={150}/>
+        </Box>
+        `}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection name="Accessibility">
+    <MainSection.Subsection
+      title="Labels"
+      description={`
+      The \`iconAccessibilityLabel\`, \`dismissButton\`, \`primaryAction\`, and \`secondaryAction\` each require a short, descriptive label for screen readers. These labels should communicate intent or visually describe what any icons might mean, such as “error”, “info” or “warning”. They should also be localized.
+
+      In the case of [Buttons](/Button), alternative text should be provided to replace vague text like "Visit" or "Learn more" with more descriptive information, like ‘Learn more about our work from home resources’.
+
+      For [IconButtons](/IconButtons), like the \`dismissButton\`, the label provided should indicate the intent, like “Dismiss this banner”.
+      `}
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
 <Callout
   type="info"
-  iconAccessibilityLabel="Info icon"
+  iconAccessibilityLabel="Info"
   title="Your business account was created!"
-  message="Apply to the Verified Merchant Program!"
-  primaryAction={{href: "https://pinterest.com", label:"Get started"}}
-  secondaryAction={{href: "https://pinterest.com", label:"Learn more"}}
+  message="Apply to the Verified Merchant Program"
+  primaryAction={{href: "https://pinterest.com", label: "Get started", accessibilityLabel:"Get started: Verified Merchant Program"}}
+  secondaryAction={{href: "https://pinterest.com", label: "Learn more", accessibilityLabel:"Learn more: Verified Merchant Program"}}
   dismissButton={{
-    accessibilityLabel: 'Dismiss banner',
+    accessibilityLabel: 'Dismiss this banner',
     onDismiss: ()=>{},
   }}
 />
-`}
-  />,
+        `}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
 );
 
 card(
-  <Example
-    name="Warning Example"
-    defaultCode={`
+  <MainSection
+    name="Localization"
+    description={`Remember to localize all Link or Button labels, as well as the \`title\` and \`message\`.`}
+  >
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
 <Callout
-  type="warning"
-  iconAccessibilityLabel="Warning icon"
-  message="This feature will be removed in two weeks."
-  primaryAction={{href: "https://pinterest.com", label:"Learn more"}}
+  type="info"
+  iconAccessibilityLabel="Info"
+  title="Ihr Geschäftskonto wurde erstellt!"
```

Comment:
German is a nice touch! 👌

Reply:
Should Verified Merchant Program be transalted too?